.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: php
 
-sudo: false
```

Comment:
yeah, I prefer being explicit too, as it does not require knowing at which date the repo started using Travis (this info is not available anywhere except in the Travis database AFAIK, and we cannot access it)